osmo_jupyter/plot.py

```diff
@@ -72,7 +72,10 @@ def get_scatter(
         },
         opacity=0.8,
         yaxis=f'y{y_axis_number}',
-        **scatter_overrides
+    )
+    scatter_kwargs.update(scatter_overrides)
```

Comment:
can you use a splat to unpack `scatter_overrides` inside of the `scatter_kwargs` definition?